examples/truffle/yarn.lock

```diff
@@ -2,61 +2,1297 @@
 # yarn lockfile v1
 
 
-"@babel/helper-module-imports@^7.12.1":
-  version "7.12.5"
-  resolved "https://registry.yarnpkg.com/@babel/helper-module-imports/-/helper-module-imports-7.12.5.tgz#1bfc0229f794988f76ed0a4d4e90860850b54dfb"
-  integrity sha512-SR713Ogqg6++uexFRORf/+nPXMmWIn80TALu0uaFb+iQIUoR7bOC7zBWyzBs5b3tBBJXuyD0cRu1F15GyzjOWA==
+"@apollo/client@^3.1.5":
+  version "3.3.11"
+  resolved "https://registry.yarnpkg.com/@apollo/client/-/client-3.3.11.tgz#125051405e83dc899d471d43b79fd6045d92a802"
+  integrity sha512-54+D5FB6RJlQ+g37f432gaexnyvDsG5X6L9VO5kqN54HJlbF8hCf/8CXtAQEHCWodAwZhy6kOLp2RM96829q3A==
```

Comment:
Looks good. My concern in the other PR was more about how this resolved to a different registry and had different integrity hashes

https://github.com/gnosis/ethcontract-rs/blob/c27204faa2920f6869c7f0cd7e8a40fb5918767f/examples/truffle/yarn.lock#L7-L8